Keyword priority (#1130)
neo4j/docs-cypher: modules/ROOT/pages/syntax/variables.adoc

.Variables are only visible in the same query part
====
Variables are not carried over to subsequent queries.
If multiple query parts are chained together using `WITH`, variables defined in one part have to be listed in the `WITH` clause to be carried over to the next part.
For more information see xref::clauses/with.adoc[WITH].
====

[IMPORTANT]
.Variables imported into a `CALL` subquery are visible in the whole subquery
====

> Even if the subquery is a multiple query parts are chained together using `WITH`, variables imported from the outer query do not have to be listed in the `WITH` clause to be visible in the next part.

Reviewer: ```suggestion
Even if the subquery consists of multiple query parts chained together using `WITH`, variables imported from the outer query do not have to be listed in the `WITH` clause to be visible in the next part.
```

> Even if the subquery consists of multiple query parts chained together using `WITH`, variables imported from the outer query do not have to be listed in the `WITH` clause to be visible in the next part.

Reviewer 2: "in the next part" tripped me up a bit.

```suggestion
Even if the subquery consists of multiple query parts chained together using `WITH`, variables imported from the outer query do not have to be listed in a `WITH` clause to be visible in subsequent parts of the subquery.
```
